playbooks/cluster_setup_network.yaml

```diff
@@ -173,14 +173,22 @@
       hostname:
         name: "{{ inventory_hostname }}"
         use: systemd
-    - name: Build hosts file
+    - name: Build hosts file (Cluster Network)
       lineinfile:
         dest: /etc/hosts
         regexp: '.*{{ item }}(| rbd)$'
         line: "{{ hostvars[item].cluster_ip_addr }} {{ item }}{% if item == inventory_hostname %} rbd{% endif %}"
         state: present
       when: hostvars[item].cluster_ip_addr is defined
       loop: "{{ groups['cluster_machines'] }}"
+    - name: Build hosts file (Public Network)
```

Comment:
You can add a comment to mention that if cluster_ip_addr is not defined, we consider that there is no cluster Network, and we use the public network for Ceph instead.

Reply:
Done